net/route/net_del_ramroute.c

```diff
@@ -91,7 +91,8 @@ static int net_match_ipv4(FAR struct net_route_ipv4_s *route, FAR void *arg)
   net_ipv4_dumproute("Comparing", route);
   ninfo("With:\n");
   ninfo("  target=%08" PRIx32 " netmask=%08" PRIx32 "\n",
-        HTONL(match->target), HTONL(match->netmask));
+        (unsigned long int)HTONL(match->target),
```

Comment:
why need change?

Reply:
Initially because the Xtensa CI is complain: "error: format specifies type 'unsigned int' but the argument has type 'unsigned long long'", then I cast to "(unsigned int)", but then ARM CI started failing because it was expecting "unsigned long int".

As you can see, there is not a cast that will work for all. This is why I asked if you know how to fix it!?

Reply:
it's better to add the cast at https://github.com/apache/nuttx/blob/master/include/netinet/in.h#L233-L241 to ensure the type in these macros same as: 
https://github.com/apache/nuttx/blob/master/include/netinet/in.h#L377-L382

Reply:
@acassis https://github.com/apache/nuttx/pull/10974 may fix your problem, please try it.